Framework/Algorithms/src/GroupToXResolution.cpp

```diff
@@ -0,0 +1,160 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#include "MantidAlgorithms/GroupToXResolution.h"
+
+#include "MantidAPI/HistogramValidator.h"
+#include "MantidAPI/WorkspaceHasDxValidator.h"
+#include "MantidDataObjects/Workspace2D.h"
+#include "MantidDataObjects/WorkspaceCreation.h"
+#include "MantidHistogramData/HistogramBuilder.h"
+#include "MantidKernel/BoundedValidator.h"
+#include "MantidKernel/CompositeValidator.h"
+
+#include <boost/math/special_functions/pow.hpp>
+
+namespace {
+namespace Prop {
+std::string const FRACTION{"FractionOfDx"};
+std::string const INPUT_WS{"InputWorkspace"};
+std::string const OUTPUT_WS{"OutputWorkspace"};
+} // namespace Prop
+constexpr double FWHM_GAUSSIAN_EQUIVALENT{0.68};
```

Comment:
Would it make sense to have this as another input in multiples of sigma?

Reply:
It is from COSMOS. I think it has some statistical backing and should not be modifiable by the user.